srcpkgs/nikto/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'nikto'
+pkgname=nikto
+version=2.2.0
+revision=1
+wrksrc=${pkgname}-${pkgname}-${version}
+depends="perl"
+short_desc="Nikto is an Open Source vulnerability scanner for web servers"
+maintainer="Samsky <samsky72@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://cirt.net/Nikto2"
+distfiles="https://github.com/sullo/${pkgname}/archive/${pkgname}-${version}.tar.gz"
+checksum=48b2ca31324f7f4100527545673c83a0566d08c2a4b2f1e1bdb299326ef52d92
+
+do_install() {
+	vmkdir usr/bin
+	vmkdir etc
+	vmkdir usr/share/${pkgname}
+	vmkdir usr/share/doc/${pkgname}
+	vcopy  program/nikto.pl  usr/bin/nikto
+	vcopy  program/replay.pl usr/bin/replay
+	vcopy  program/plugins usr/share/${pkgname}
+	vcopy  program/templates usr/share/${pkgname}
+	vcopy  program/databases usr/share/${pkgname}
+	vman   program/docs/${pkgname}.1
+	vdoc   program/docs/nikto_manual.html
+	vconf  ${FILESDIR}/nikto.conf
```

Comment:
Instead of vendoring the config, why not take the default config at `program/nikto.conf.default`